apps/s_time.c

```diff
@@ -179,6 +183,18 @@ int s_time_main(int argc, char **argv)
         case OPT_SSL3:
             max_version = SSL3_VERSION;
             break;
+        case OPT_TLS1:
+            max_version = TLS1_VERSION;
```

Comment:
Wouldn't you also need to set min_version?

Reply:
Actually I am wondering why we need to introduce the `min_version`, it seems we could just call `SSL_CTX_set_min_proto_version` with the `max_version` variable in this scenario. (or rename it to something like `version`)